code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -193,6 +193,23 @@
 		holder.remove_reagent("capsaicin", 2)
 	..()
 
+/datum/reagent/consumable/drink/milk/on_skeleton_life(mob/living/carbon/human/H)
```

Comment:
This is the same effect they already had, so I won't be changing this. Just commenting on that here since you technically included this in the "request changes" review. I did not alter and do not plan to alter any of the numbers here in this PR since that's balance stuff and this is not meant to be a balance PR.